tests/dummy/app/templates/components/expanded-row.hbs

```diff
@@ -7,9 +7,30 @@
     <p>{{@row.bio}}</p>
 
     <div class="user-actions">
-      <a class="fa fa-envelope" href="mailto:{{@row.email}}"></a>
-      <a class="fa fa-facebook" href="#"></a>
-      <a class="fa fa-twitter" href="#"></a>
```

Comment:
Why did we remove font-awesome here? I had updated font-awesome, so we should be able to still use it.

Reply:
New addon is official font-awesome add-on, not a community add-on. It does not support using class names. You need to use their `FaIcon` component or  the CDN.

I believe you can create a "kit' which is mini CDN but that requires managing something through their servers.  Long term we should try to remove the dependency.  Everyone using ELT and cuurent addon have both both addons in their project.

Reply:
@maxwondercorn yes, I know it is an official font-awesome addon, I updated to it. The class names should still work. I converted a bunch of `asc` and `desc` to `up` and `down` and they worked.

Reply:
That's interesting.  I had no icons displayed.  Maybe I still had asc/desc instead of up/down in the names but none of the others were showing either. IIRC my main app had the same issue and I switched to the fa component.

I'll pull all the changes back in and will undo changes as needed